.buildkite/pipeline.yml

```diff
@@ -22,45 +22,114 @@ steps:
       slurm_mem: 32G
       slurm_gpus: 1
       slurm_cpus_per_task: 8
-      slurm_gpus: 1
       slurm_ntasks: 1
       slurm_gpus_per_task: 1
     env:
       JULIA_NUM_PRECOMPILE_TASKS: 8
 
   - wait
 
-  - label: "Run JRA55 tests"
+  - label: "CPU JRA55 tests"
     key: "tests_jra55"
     env:
-      CUDA_VISIBLE_DEVICES: "-1"
       TEST_GROUP: "jra55"
     commands:
       - "julia --project -e 'using Pkg; Pkg.test()'"
+    agents:
+      slurm_mem: 32G
+      slurm_cpus_per_task: 8
+      slurm_ntasks: 1
 
-  - label: "Run ECCO tests"
+  - label: "CPU ECCO tests"
     key: "tests_ecco"
     env:
-      CUDA_VISIBLE_DEVICES: "-1"
       TEST_GROUP: "ecco"
     commands:
       - "julia --project -e 'using Pkg; Pkg.test()'"
+    agents:
+      slurm_mem: 32G
+      slurm_cpus_per_task: 8
+      slurm_ntasks: 1
 
-  - label: "Run surface flux tests"
+  - label: "CPU surface flux tests"
     key: "tests_fluxes"
     env:
-      CUDA_VISIBLE_DEVICES: "-1"
       TEST_GROUP: "fluxes"
     commands:
       - "julia --project -e 'using Pkg; Pkg.test()'"
+    agents:
+      slurm_mem: 32G
+      slurm_cpus_per_task: 8
+      slurm_ntasks: 1
 
-  - label: "Run bathyemtry tests"
+  - label: "CPU bathymetry tests"
```

Comment:
Is it best to use a separate job for CPU vs GPU tests, or just do both CPU and GPU tests in one job?